exercises/kindergarten-garden/uKindergartenGardenTest.pas

```diff
@@ -0,0 +1,145 @@
+unit uKindergartenGardenTest;
+
+interface
+uses
+  DUnitX.TestFramework, uKindergartenGarden;
+
+type
+  [TestFixture]
+  TPartialGardenTest = class(TObject)
+  private
+    procedure CompareStringArrays(ExpectedArray, ActualArray : TArray<string>);
+  public
+    [Test]
+//  [Ignore('Comment the "[Ignore]" statement to run the test')]
+    procedure garden_with_single_student;
+
+    [Test]
+    [Ignore]
+    procedure different_garden_with_single_student;
+
+    [Test]
+    [Ignore]
+    procedure garden_with_two_students;
+
+    [Test]
+    [Ignore]
+    procedure multiple_students_for_the_same_garden_with_three_students_second_students_garden;
+
+    [Test]
+    [Ignore]
+    procedure multiple_students_for_the_same_garden_with_three_students_third_students_garden;
+
+    [Test]
+    [Ignore]
+    procedure first_students_garden;
+
+    [Test]
+    [Ignore]
+    procedure second_students_garden;
+
+    [Test]
+    [Ignore]
+    procedure second_to_last_students_garden;
+
+    [Test]
+    [Ignore]
+    procedure last_students_garden;
+  end;
+
+implementation
+
+procedure TPartialGardenTest.CompareStringArrays(ExpectedArray,
+  ActualArray: TArray<string>);
+var
+  i: integer;
+begin
+  Assert.AreEqual(Length(ExpectedArray), Length(ActualArray));
+  for i := Low(ExpectedArray) to High(ExpectedArray) do
+    Assert.AreEqual(ExpectedArray[i], ActualArray[i]);
+end;
+
+procedure TPartialGardenTest.different_garden_with_single_student;
+var Garden : TGarden;
```

Comment:
move `Garden: TGarden` down to next line, indent in 2 spaces.  Repeat for all tests.  Just going for uniform look on test suite.